src/app-server.ts

```diff
@@ -30,31 +30,31 @@ function isReachable(port: number, options: { timeout: number } = { timeout: 100
       execution.resume(true);
     });
 
-    return () => { socket.destroy() };
+    execution.ensure(() => socket.destroy());
```

Comment:
`atExit()` is `ensure()` in the current version of effection.

caveats:

1. this name [will likely change](https://github.com/thefrontside/effection.js/issues/66)
2. It is not currently in the typings for effection (but will be at some point)